src/main/java/fact/datacorrection/DrsCalibration.java

```diff
@@ -256,11 +279,21 @@ public Data process(Data data) {
 				triggerOffsetPos = pixel * drsTriggerOffsetMean.length / 1440
 						+ slice;
 
-				vraw = data[pos] * dconv;
-				vraw -= drsBaselineMean[offsetPos];
-				vraw -= drsTriggerOffsetMean[triggerOffsetPos];
-				vraw /= drsGainMean[offsetPos];
-				vraw *= 1907.35;
+				if (!reverse) {
```

Comment:
shouldn't `reverse` be handed over to the function. I know its a global variable. But i think, what it means for the function, is more obvious if its handed over